.travis.yml

```diff
@@ -21,3 +21,10 @@ script:
 cache:
   directories:
   - debian/node_modules
+
+deploy:
+  provider: packagecloud
+  repository: "headset-electron"
+  username: "headsetapp"
+  token: "${PACKAGECLOUD_TOKEN}"
+  dist: "ubuntu/precise"
```

Comment:
why did you choose `precise` as the target? I know it doesn't really matter much, but I would suggest changing it to at least `trusty`. Travis is running on `trusty` so it would make sense to deploy on that distro. It would be even better to point to the latest LTS `xenial` but we would have to check manually instead of Travis. Again, there shouldn't be a problem because headset only depends on `dbus` and nothing else.